src/Microsoft.ML.Tokenizers/PreTokenizer/Roberta.cs

```diff
@@ -18,7 +18,7 @@ public sealed partial class RobertaPreTokenizer : PreTokenizer
         /// </summary>
         public static RobertaPreTokenizer Instance { get; } = new RobertaPreTokenizer();
 
-        private const string PretokenizePattern = @"'s|'t|'re|'ve|'m|'ll|'d| ?\p{L}+| ?\p{N}+| ?[^\s\p{L}\p{N}]+|\s+(?!\S)|\s+";
+        private const string PretokenizePattern = /*lang=regex*/ @"'(?:[sdmt]|re|ve|ll)| ?\p{L}+| ?\p{N}+| ?[^\s\p{L}\p{N}]+|\s+(?!\S)|\s+";
```

Comment:
Can we add a reference to the upstream location that we sync these too?

Reply:
Seems like that is https://github.com/openai/tiktoken/blob/main/tiktoken_ext/openai_public.py?

Reply:
> Seems like that is https://github.com/openai/tiktoken/blob/main/tiktoken_ext/openai_public.py?

Yes